Essentials/src/com/earth2me/essentials/commands/Commandignore.java

```diff
@@ -44,7 +45,15 @@ protected void run(final Server server, final User user, final String commandLab
                 user.sendMessage(tl("unignorePlayer", player.getName()));
             } else {
                 user.setIgnoredPlayer(player, true);
-                user.sendMessage(tl("ignorePlayer", player.getName()));
+                if (args.length > 1) {
```

Comment:
Only requires one argument
```suggestion
                if (args.length > 0) {
```

Reply:
The block here processes a second argument if given, to run the extended functionality, dealing with the optional timer argument.